### PR TITLE
@ #11 | fix bug failed to sync created files from Linux host

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #11 
please check this PR! Thank you!

### CODE REVIEW

File: lib/teracy-dev-v05-compat/config/rsync_recovery.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+      def rsync_cmd
+        cmd = ''
+
+        if Vagrant::Util::Platform.linux?
+          compatitive_version = ">= 2.2.3"
+
+          unless require_version_valid?(compatitive_version)
+            @logger.warn("Please use vagrant `#{compatitive_version}` to fix the problem: vagrant crashed, can not be recovered.")
+            @logger.warn("See more at: https://github.com/hashicorp/vagrant/issues/10460")
```
Comment (hoatle) [inline lib/teracy-dev-v05-compat/config/rsync_recovery.rb:70]: in this case, use `gatling-rsync-auto`

**Review by hoatle — CHANGES_REQUESTED** [review]
@hieptranquoc please check my comments.

**Comment by hieptranquoc** [comment]
@hoatle I applied your comment! Please check this PR again! Thank you!

**Comment by hoatle** [comment]
need @phuonglm to review.

**Comment by hieptranquoc** [comment]
will be fixed when teracy-dev-essential v0.3.0 is released!

**Pull request closed without merging**